src/it/scala/org/apache/spark/sql/SFDataFrameWindowFramesSuite.scala

```diff
@@ -1,14 +1,358 @@
 package org.apache.spark.sql
 
+import org.apache.spark.sql.expressions.Window
+import org.apache.spark.sql.functions._
 import org.apache.spark.sql.snowflake.{SFQueryTest, SFTestData, SFTestSessionBase}
 
 class SFDataFrameWindowFramesSuite
-    extends DataFrameWindowFramesSuite
+    extends SFQueryTest
     with SFTestSessionBase
-    with SFQueryTest
     with SFTestData {
 
+  import SFTestImplicits._
+
   override def spark: SparkSession = getSnowflakeSession()
 
   override protected def blackList: Seq[String] = Seq.empty
+
```

Comment:
To reviewers:
`SFDataFrameWindowFramesSuite` and `SFDataFrameWindowFunctionsSuite` don't actually run with snowflake database. I copy all `test` from `DataFrameWindowFramesSuite` and `DataFrameWindowFunctionsSuite`, and then make them run with snowflake.